### ISSUE TITLE
Why the adative learning scheduler?

### ISSUE THREAD

**Issue by ShinyMcShine** [Shawn-Shan/trapdoor#4]
Mainly a question in training design, why use an adaptive training scheduler based on epoch when your model is using an Adam optimizer does this as well?

**Comment by Shawn-Shan** [comment]
For adaptive learning rate, it is common practice to train large models to gradually reduce the learning rate.

**Comment by ShinyMcShine** [comment]
Thanks for replying back. 

Is this to keep the adam optimizer in check so that its adaptation does adjust the learning rate too much? Why have the same learning right after 10 epochs? From 10 to 40 the learning rate does not change.
